lib/chef/dsl/resources.rb

```diff
@@ -0,0 +1,22 @@
+class Chef
+  module DSL
+    #
+    # Module containing a method for each globally declared Resource
+    #
+    # Depends on declare_resource(name, created_at, &block)
+    #
+    # @api private
+    module Resources
+      def self.add_resource_dsl(dsl_name)
+        module_eval <<-EOM, __FILE__, __LINE__+1
```

Comment:
Can we use parens to be consistent with the rest of the code? This and driving are the only things I don't like "Seattle style" :stuck_out_tongue_winking_eye: